sort.py

```diff
@@ -0,0 +1,22 @@
+
+def sort(array):
```

Comment:
could we rename to clarify how we are sorting? highest to lowest? lowest to highest?